libadcc/HartreeFockSolution_i.hh

```diff
@@ -41,14 +41,6 @@ namespace libadcc {
  */
 class HartreeFockSolution_i {
  public:
-  /** \name System information */
-  ///@{
-  /** Fill a buffer with nuclear multipole data for the nuclear multipole of
-   *  given order. */
-  virtual void nuclear_multipole(size_t order, scalar_type* buffer,
```

Comment:
I would definitely not delete this from the base class because nuclear dipoles should be a requirement for the backend. I would rather distinguish between `nulcear_dipole` and `nuclear_multipole` and the latter could then be implemented on the Python side only.